docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml

```diff
@@ -160,7 +160,50 @@ methods:
           type: string
       return:
         type: '<xref uid="ExcelScript!ExcelScript.Shape:interface" />'
-        description: ''
+        description: |-
+
+
+          #### Examples
+
+          ```typescript
+          /**
+           * This sample copies an image from a URL. This could be used to copy photos that a colleague stored in a shared folder to a related workbook.
+           */
+          async function main(workbook: ExcelScript.Workbook) {
+            // Fetch the image from a URL.
+            const link = "https://github.com/OfficeDev/office-scripts-docs/blob/master/docs/images/git-octocat.png";
+            const response = await fetch(link);
```

Comment:
This fetch call fails for me. If I change to this link it does work: https://raw.githubusercontent.com/OfficeDev/office-scripts-docs/master/docs/images/git-octocat.png